simpleLooping/binaryListToString.py

```diff
@@ -2,7 +2,7 @@
 #       Despite their differences, both systems assign values to each place number.
 #
 #       base 10: 1011 = 1 * 10^0 + 1 * 10^1 + 0 * 10^2 + 1 * 10^3
-#                     = 1 * 1    + 1 * 10   + 0 * 100  + 1 * 1000
+#                     = 1 * 1    + 1 * 10   + 0 * 100  + 1 * 1000   
```

Comment:
Please fix this one. Did you "git diff --staged" before committing?

Reply:
I did saw this, but It was as problem set sample so I was pass it, but I can fix it if it an issue for you.

Reply:
Please review this one.